packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/edge/CustomEdge.tsx

```diff
@@ -38,7 +38,7 @@ export const CustomEdge = memo(
       targetPosition,
     });
 
-    const { beginLabel, endLabel, centerLabel } = data;
+    const { beginLabel, endLabel, centerLabel } = data || {};
```

Comment:
data can be null / undefined?

Reply:
yes, data is optional in EdgeProps

Reply:
https://github.com/wbkd/react-flow/discussions/1923

Indeed that's a bit odd but ok